main.go

```diff
@@ -54,32 +57,111 @@ func main() {
 	if err := stepconf.Parse(&config); err != nil {
 		fail("Issue with input: %s", err)
 	}
+
 	if err := validateGoTemplate(config.PublicInstallPageMapFormat); err != nil {
 		fail("PublicInstallPageMapFormat - %s", err)
 	}
 
 	stepconf.Print(config)
-
 	fmt.Println()
-
 	log.SetEnableDebugLog(config.DebugMode)
 
 	absDeployPth, err := pathutil.AbsPath(config.DeployPath)
 	if err != nil {
 		fail("Failed to expand path: %s, error: %s", config.DeployPath, err)
 	}
 
-	filesToDeploy := []string{}
-
 	tmpDir, err := pathutil.NormalizedOSTempDirPath("__deploy-to-bitrise-io__")
 	if err != nil {
 		fail("Failed to create tmp dir, error: %s", err)
 	}
 
-	// Collect files to deploy
+	filesToDeploy, err := collectFilesToDeploy(absDeployPth, config, tmpDir)
+	if err != nil {
+		fail(err)
+	}
+	clearedFilesToDeploy := clearDeployFiles(filesToDeploy)
+	fmt.Println()
+	log.Infof("List of files to deploy")
+	logDeployFiles(clearedFilesToDeploy)
```

Comment:
this method has logs outside of main

Reply:
The method itself is for logging, the name indicates it.